include/inviwo/core/util/stdextensions.h

```diff
@@ -33,6 +33,7 @@
 #include <inviwo/core/common/inviwocoredefine.h>
 #include <inviwo/core/util/glm.h>
 #include <inviwo/core/util/foreacharg.h>
+#include <inviwo/core/util/hashcombine.h>
```

Comment:
Should update the changelog and state this change

Reply:
It does not break any code.

Reply:
Ok, not ours at least ;)

Reply:
it was in stdextensions before, and now stdextensions include this one. So it will not break _any_ code as I said.